src/Workspaces/Core/Portable/Utilities/SpellChecker.cs

```diff
@@ -14,14 +13,12 @@ namespace Roslyn.Utilities
     /// Explicitly a reference type so that the consumer of this in <see cref="BKTree"/> can safely operate on an
     /// instance without having to lock to ensure it sees the entirety of the value written out.
     /// </summary>>
-    internal sealed class SpellChecker(Checksum checksum, BKTree bKTree) : IObjectWritable, IChecksummedObject
```

Comment:
we were doing nothing with this, except writing/reading it from sql (never checking it or anything).  just removed.